### DISCUSSION TITLE
[Github Projects Beta] How can I group a view in a project by milestone and sort it based on the time period.

### DISCUSSION THREAD

**Discussion by ramith** [community/community#29760] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi All,
 I have a private Github Project (Beta) created for issues in a repository. and in a view I have grouped these issues by milestone.
The problem I'm having is sort order for milestone seems to be in alphabetical order. How can i make it appear based on the time?

For example, consider my milestone are:
`["May, 2022", "June, 2022", "July, 2022", "August, 2022", "September, 2022", "October, 2022"]`

So in the view they appear in following order:
`["August, 2022", "July, 2022", "June, 2022", "May, 2022", "October, 2022", "September, 2022"]`

Any ideas?

**Comment by philip-gai** [comment, marked as answer]
In this case I would suggest using the `YYYY-MM` date format which will sort nicely. For example: `["2022-05", "2022-06", "2022-07", "2022-08", "2022-09", "2022-10"]`

For milestones with non-date related titles, prefixing with a number helps i.e. `01. Planning`, `02. Prototype`, etc.

**Comment by ramith** [comment]
I changed the labels as @philip-gai suggested and it works! Thanks you!
